modules/check_fastq.py

```diff
@@ -68,14 +68,61 @@ def sort_by_gc(seqs: dict, gc_bounds: Union[int, tuple[int]] = (0, 100)):
     return unsuitable_by_gc
 
 
-def fastq_check(seqs: dict, length_bounds: Union[int, tuple[int]] = (0.2 ** 32),
+def fastq_to_dict(input_path: str) -> dict:
```

Comment:
1) Название функции должны быть глаголом
2) Что это функция делает? Можно было бы сказать что она "конвертирует" фастку-файл в питоновский словарь. Но все таки по людски это называется чтение файлов
3) И нам тут в целом даже не так важно во что она его читает - в словарь, в список или в пандас датафрейм

Исходя из этого всего в совокупности идеальное имя тут:
```suggestion
def read_fastq(input_path: str) -> dict:
```

Reply:
А то что она читает именно в словарь у тебя кстати вполне правильно отмечено в аннотации типов! Этого достаточно